### PR TITLE
add feedstock for pynion

### PR DESCRIPTION
**Pull request by hartb**

### CODE REVIEW

File: conda-recipes/pynion-feedstock/recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  name: pynion
+  version: {{ version }}
+
+source:
+  url: https://github.com/jaumebonet/pynion/archive/{{ version }}.tar.gz
+  sha256: 6798f9359931fa65490e722e7958ed218c889c26e8f48fe052976391003df37a
+
+requirements:
+  host:
+    - python >=2.7.0,<2.8.0a0
```
Comment (wdirons) [inline conda-recipes/pynion-feedstock/recipe/meta.yaml]: I would remove the python version requirement and let people use with python 3. Looking at setup.py it should work with python 3.

File: conda-recipes/pynion-feedstock/.ci_support/linux_noarch.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+c_compiler:
+- gcc
+c_compiler_version:
+- '8'
+channel_sources:
+- conda-forge,defaults
```
Comment (wdirons) [inline conda-recipes/pynion-feedstock/.ci_support/linux_noarch.yaml]: It doesn't appear to need conda-forge as a source, as both packages required are available in main,

File: conda-recipes/pynion-feedstock/recipe/meta.yaml (hunk 1 above)
Comment (hartb) [reply conda-recipes/pynion-feedstock/recipe/meta.yaml]: There's some other code in pynion (and StrBioInfo to follow) doesn't actually work in py3, and would need porting effort. I thought about adding auto-generated patches using `2to3`and only applying in the python 3 case, but figure that can be added later if there's any interest

File: conda-recipes/pynion-feedstock/.ci_support/linux_noarch.yaml (hunk 2 above)
Comment (hartb) [reply conda-recipes/pynion-feedstock/.ci_support/linux_noarch.yaml]: Agree. I'd seen the same. Decided there was no harm in leaving it just for better commonality between yaml files here and for other noarch packages.

**Pull request merged**